Common/Product/SharedProject/ProjectNode.cs

```diff
@@ -1593,6 +1597,16 @@ internal enum EventTriggering {
             }
         }
 
+        private void LogProjectLoadTelemetry(string eventName) {
+            DataPointCollection properties = new DataPointCollection();
+            // Enclose guid in braces
+            properties.Add(TelemetryProperties.ProjectGuid, this.ProjectIDGuid.ToString("B"));
+            properties.Add(TelemetryProperties.ProjectFlavorGuid, this.ProjectGuid.ToString("B"));
+            properties.Add(TelemetryProperties.ProjectType, this.ProjectType);
+
+            NodejsPackage.Instance.TelemetryLogger.ReportEvent(eventName, properties);

```

Comment:
The properties set here seem like they should be set pretty much every time we log telemetry involving a project, so there should be a more centralized method for it.